mainwindow.h

```diff
@@ -43,6 +45,7 @@ class MainWindow : public QMainWindow
 
 	private:
 		QListWidgetItem *getTwitchChatMessage(QString username, QString message);
+		QPushButton *btnStart;
```

Comment:
btnStart is not really a good name imho

Reply:
I'm not good at giving names. How should it be called? 🙃